drivers/include/bmp180.h

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright (C) 2016 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_bmp180 BMP180
+ * @ingroup     drivers_sensors
+ * @brief       Device driver interface for the BMP180 sensor
+ * @{
+ *
+ * @file
+ * @brief       Device driver interface for the BMP180 sensor.
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ */
+
+#ifndef BMP180_H_
+#define BMP180_H_
+
+#include "periph/i2c.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name BMP180 I2C address
+ * @{
+ */
+#define BMP180_ADDR                   (0x77) // 7 bits address
+/** @} */
+
+/**
+ * @name BMP180 registers
+ * @{
+ */
+#define BMP180_REGISTER_ID            (0xD0)
+#define BMP180_REGISTER_CONTROL       (0xF4)
+#define BMP180_REGISTER_DATA          (0xF6)
+#define BMP180_TEMPERATURE_COMMAND    (0x2E)
+#define BMP180_PRESSURE_COMMAND       (0x34)
+#define BMP180_CALIBRATION_AC1        (0xAA)
+/** @} */
```

Comment:
Look at some other drivers which use a pattern like `drivers/include/foo.h` and `drivers/foo/include/foo-internal.h`